commands.sh

```diff
@@ -98,9 +98,16 @@ tdrs-lint-backend() {
     tdrs-compose-backend run --rm web bash -c "flake8 . && if [ $? -eq 0 ]; then echo 'Flake8 linter found no issues'; fi"
 }
 
+# create docker network for tdrs if it doesn't exist
+ tdrs-docker-net() {
+     docker network inspect external-net >/dev/null 2>&1 \
```

Comment:
my only concern here is that this _forces_ usage of the frontend `docker-compose.yml` during local development, whereas before we had the option to only run the backend in docker, and run the frontend with `npm run start`. 

the frontend dockerfile performs a development build of the frontend files and serves them via nginx, so we'd lose the hot reload and error reporting functionality of `npm run start` when developing locally. if there's a way around that, i'd like to know because i think it could make future frontend tickets a little more cumbersome to work on.

Reply:
You can use docker-compose as before, the backend is still available on port 8080, so by only changing the backendURL in frontend env variables, you can build frontend locally and connect it to your backend.